.github/workflows/ci.yml

```diff
@@ -0,0 +1,32 @@
+name: CI
+
+on:
+  schedule:
+      # once per day
+    - cron: 0 0 * * *
+  push:
+    branches:
+      - development
+      - master
+  pull_request:
+    branches:
+      - development
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node-version: [14.x]
```

Comment:
Since this is new config I think we should run on 16.x
From https://nodejs.org/en/download/: `Latest LTS Version: 16.13.0`
https://github.com/actions/setup-node: supports 16